cockatrice/src/tab_supervisor.cpp

```diff
@@ -559,3 +561,19 @@ bool TabSupervisor::getAdminLocked() const
         return true;
     return tabAdmin->getLocked();
 }
+
+void TabSupervisor::processNotifyUserEvent(const Event_NotifyUser &event)
+{
+    QString notificationTitle("Notification");
+    QString notificationMessage;
+    switch ((Event_NotifyUser::NotificationType) event.type()) {
+        case Event_NotifyUser::PROMOTIONSUCCESS: QMessageBox::information(this, tr("Success"), tr("Successfully promoted user.")); break;
```

Comment:
I think we should omit that for now. Let's make the scope of this PR just to add the protocol ability for single actions + ui to do it